tools/mkpkg/update_retroplayer-addons

```diff
@@ -73,6 +73,14 @@ mkdir -p "${TMPDIR}"
 
 . "${MY_DIR}/update_common_functions"
 
+get_gh_token
```

Comment:
this results in an error - function is called `get_ghtoken` in update_common_functions

Better change `get_ghtoken` to `get_gh_token" in update_common_functions and update_binary_addons though, as that is more readable than `get_ghtoken`

Reply:
was history related, fixed